### ISSUE TITLE
cross compiling from linux to windows 

### ISSUE THREAD

**Issue by matthewashton-k** [burtonageo/cargo-bundle#97]
Hey, Is it possible to create an msi installer file on my ubuntu laptop, I installed the toolchain and linker to set up cross compilation, and I can successfully create a .exe on my computer, and I was wondering if it was possible to cross compile and make a .msi from my linux laptop. I tried it, but it failed with: 
```
warning: MSI bundle support is still experimental.
    Bundling cof_application.msi
error: Failed to collect resource file information
  Caused by: No such file or directory (os error 2)
```

Does anyone know if this is possible?

**Comment by yuhr** [comment]
No, it's not possible as Windows support is broken for now (#77). It fails even when run on Windows.
